objects/src/builder/mod.rs

```diff
@@ -59,8 +119,8 @@ impl<T: Rng> AccountIdBuilder<T> {
         self
     }
 
-    pub fn code_root(&mut self, code_root: Digest) -> &mut Self {
-        self.code_root = code_root;
+    pub fn code<C: AsRef<str>>(&mut self, code: C) -> &mut Self {
+        self.code = code.as_ref().to_string();
```

Comment:
Changed it to receive the code instead of the digest. It should be easier to use this way, because the user doesn't have to instantiate the `assembler` themselves.